web_accounts_list.json

```diff
@@ -6,14 +6,15 @@
               "Creative Commons, PO Box 1866, Mountain View, CA 94042, USA."],
   "authors" : ["WebBreacher","Munchko","L0r3m1p5um","lehuff","janbinx","bcoles","Sector035",
               "arnydo","mccartney","salaheldinaz","camhoff","jocephus","swedishmike","soxoj",
-              "jspinel","ef1500","zewen","jocejocejoe"],
+              "jspinel","ef1500","zewen","jocejocejoe", "P3run"],
   "categories" : ["art","blog","business","coding","dating","finance","gaming","health",
                 "hobby","images","misc","music","news","political","search","shopping","social",
                 "tech","video","XXXPORNXXX"],
   "sites" : [
      {
        "name" : "7cup",
        "check_uri" : "https://www.7cups.com/@{account}",
+       "cookies": {"admin": "true"},

```

Comment:
Oops. I think you might have missed removing this `cookies` tag. Can you remove it?

Reply:
I thought so. Really appreciate your additions here.